<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="GLWidget" name="GLwidget" native="true">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>310</width>
        <height>310</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>16777215</height>
       </size>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>220</width>
         <height>421</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QGroupBox" name="renderingBox">
          <property name="minimumSize">
           <size>
            <width>190</width>
            <height>130</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>170</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Rendering</string>
          </property>
          <widget class="QCheckBox" name="smoothShadeCheckBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>131</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Smooth Shade</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QCheckBox" name="wireframeCheckBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>151</width>
             <height>22</height>
            </rect>
           </property>
           <property name="text">
            <string>Show Wireframe</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="lanternBox">
          <property name="title">
           <string>Lantern Controls</string>
          </property>
          <widget class="QPushButton" name="optimizeButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>170</y>
             <width>97</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string>Test</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="GLWidget2D" name="GLwidget2D" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>440</y>
         <width>300</width>
         <height>300</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVViewer">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_OBJ"/>
    <addaction name="actionSave_Simulation"/>
    <addaction name="actionExit"/>
    <addaction name="actionSchwarz_Lantern"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionReset_Camera"/>
    <addaction name="actionTake_Screenshot"/>
    <addaction name="actionExport_OBJ"/>
    <addaction name="actionImport_OBJ"/>
   </widget>
   <addaction name="menuVViewer"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_OBJ">
   <property name="text">
    <string>Load Simulation</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionReset_Camera">
   <property name="text">
    <string>Reset Camera</string>
   </property>
  </action>
  <action name="actionTake_Screenshot">
   <property name="text">
    <string>Take Screenshot</string>
   </property>
  </action>
  <action name="actionSchwarz_Lantern">
   <property name="text">
    <string>Schwarz Lantern</string>
   </property>
  </action>
  <action name="actionExport_OBJ">
   <property name="text">
    <string>Export 3D Mesh</string>
   </property>
  </action>
  <action name="actionSave_Simulation">
   <property name="text">
    <string>Save Simulation</string>
   </property>
  </action>
  <action name="actionImport_OBJ">
   <property name="text">
    <string>Import 3D Mesh</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLWidget2D</class>
   <extends>QWidget</extends>
   <header>glwidget2d.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
